_issues/but-hillary-wants-my-guns.md

```diff
@@ -15,11 +15,11 @@ Q: What do those solutions look like?
 
-+ __Hold accountable irresponsible gun dealers.__ Because of the Protection of Lawful Commerce in Arms Act, victims of gun violence [cannot hold liable](https://en.wikipedia.org/wiki/Protection_of_Lawful_Commerce_in_Arms_Act) negligent gun dealers and manufacturers. Hillary will work to [repeal](http://www.courant.com/politics/capitol-watch/hc-hillary-clinton-sandy-hook-guns-story.html) this law and [revoke](http://www.nbcnews.com/meet-the-press/hillary-clinton-unveil-plan-major-new-gun-restrictions-n438361) the licenses of gun dealers who knowingly sell to traffickers and straw purchasers (individuals who buy guns on behalf of violent felons).  
++ __Hold irresponsible gun dealers accountable.__ Because of the Protection of Lawful Commerce in Arms Act, victims of gun violence [cannot hold liable](https://en.wikipedia.org/wiki/Protection_of_Lawful_Commerce_in_Arms_Act) negligent gun dealers and manufacturers. Hillary will work to [repeal](http://www.courant.com/politics/capitol-watch/hc-hillary-clinton-sandy-hook-guns-story.html) this law and [revoke](http://www.nbcnews.com/meet-the-press/hillary-clinton-unveil-plan-major-new-gun-restrictions-n438361) the licenses of gun dealers who knowingly sell to traffickers and straw purchasers (individuals who buy guns on behalf of violent felons).  
 
-+ __Keep guns out of the hands of violent criminals and those with severe mental illness.__ Hillary will support laws preventing domestic abusers from buying and owning gun. She will make it a federal crime to intentionally buy a gun for a person prohibited from owning one. She will close [loopholes](http://www.nytimes.com/interactive/2016/01/06/us/how-people-with-mental-illness-are-able-to-obtain-guns.html?_r=0) that allow people suffering from severe mental illness to purchase and own guns. And she will work to [ban assault weapons.](https://www.washingtonpost.com/news/post-politics/wp/2016/06/13/clinton-calls-for-a-new-assault-weapons-ban-12-years-after-the-last-one-expired/)
++ __Keep guns out of the hands of violent criminals and those with severe mental illness.__ Hillary will support laws preventing domestic abusers from buying and owning gun. She will make it a federal crime intentionally to buy a gun for a person prohibited from owning one. She will close [loopholes](http://www.nytimes.com/interactive/2016/01/06/us/how-people-with-mental-illness-are-able-to-obtain-guns.html?_r=0) that allow people suffering from severe mental illness to purchase and own guns. And she will work to [ban assault weapons.](https://www.washingtonpost.com/news/post-politics/wp/2016/06/13/clinton-calls-for-a-new-assault-weapons-ban-12-years-after-the-last-one-expired/)
```

Comment:
This one doesn't seem right. `to intentionally buy a gun` vs. `intentionally to buy a gun`.